nexus-writer/src/main.rs

```diff
@@ -60,15 +65,18 @@ struct Cli {
     #[clap(long)]
     alarm_topic: String,
 
+    /// Topic to publish frame assembled event packets to
     #[clap(long)]
     frame_event_topic: String,
 
     #[clap(long)]
     file_name: PathBuf,
```

Comment:
What is this?

Reply:
What do you mean by this? Should I expand on what is meant by a topic in this context?

Reply:
No, `file_name` is still undocumented.

Reply:
Oops sorry, I thought you were referring to L68.